src/utils/formatNumbers.ts

```diff
@@ -59,3 +60,13 @@ export const formatTransactionAmount = (num: number | undefined | null, maxDigit
   }
   return `${Number(num.toFixed(2)).toLocaleString(DEFAULT_LOCALE, { minimumFractionDigits: 2 })}`
 }
+
+const MAX_AMOUNT_STR_LENGTH = 9
+
+export function formatReviewSwapCurrencyAmount(amount: CurrencyAmount<Currency>, maxLength?: number): string {
```

Comment:
nit: considering that maxlength is never used as an argument in any of your implementations is making it an argument necessary?